### DISCUSSION TITLE
v1.7.3

### DISCUSSION THREAD

**Discussion by jmoralez** [Nixtla/neuralforecast#1051] [Announcements]
## New Features

- [FEAT] ISQF @elephaint (#1019)
- [FEAT] - Add SOFTS model @marcopeix (#1024)
- [FEAT] Add option to support user defined learning rate scheduler for NeuralForecast Models @JQGoh (#998)
- [FEAT] Implicit Quantile Networks @elephaint (#1007)

## Bug Fixes

- use assign argument if available in nn.Module.load_state_dict @jmoralez (#1032)
- update min_size in TimeSeriesDataset.append @jmoralez (#1033)
- fix num_tasks in spark integration @jmoralez (#1028)

## Documentation

- fix: add tsmixer tutorial to sidebar @AzulGarza (#978)
- Update models in the README @candalfigomoro (#946)

## Enhancement

- suppress warning when saving hyperparameters in base auto @jmoralez (#1034)
- automatically set refit_with_val when early stopping is enabled @jmoralez (#1031)

<hr /><em>This discussion was created from the release <a href='https://github.com/Nixtla/neuralforecast/releases/tag/v1.7.3'>v1.7.3</a>.</em>
